scaleway/resource_function_cron_test.go

```diff
@@ -77,6 +78,73 @@ func TestAccScalewayFunctionCron_Basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckScalewayFunctionCronExists(tt, "scaleway_function_cron.main"),
 					resource.TestCheckResourceAttr("scaleway_function_cron.main", "schedule", "0 0 * * *"),
+					resource.TestCheckResourceAttr("scaleway_function_cron.main", "name", "tf-tests-cron-basic"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccScalewayFunctionCron_NameUpdate(t *testing.T) {
```

Comment:
This test of update could be done in the basic test.
In the future, if we add a description attribute, it can fit in basic too. As long as the field is simple, there is no need for a seperate test